### PR TITLE
feat: add staff to db

### PR DESCRIPTION
**Pull request by jtowarnicka**
## Description

added node-scheduler to dependencies
created model schema
cron job checks every night at midnight whether there're some changes on mfi staff page to update database

## Clickup Ticket
[https://sharing.clickup.com/26492745/t/h/862j3bvjx/L9FS96DBEOZ1JBJ](url)

### CODE REVIEW

File: src/Controllers/staff.controller.ts [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
-        res.status(staff.status).send(staff.message);
-        return;
+    try {
+        const staff = await StaffModel.find({}, { _id: 0, __v: 0 });
+        res.status(200).json(staff);
+    } catch (error) {
+        res.status(500).json({
+            status: 500,
+            message: `There's an ${error}. Please try again.`,
+        });
```
Comment (wgola) [inline src/Controllers/staff.controller.ts:18]: Add `return` before `res.status...` and empty line before `return`

**Review by wgola — APPROVED** [review]
LGMT, I suggest only one change:

File: src/Services/staffScraper.service.ts [hunk 2]
```diff
@@ -12,11 +12,23 @@ const facultyMemberScraper = async (
...
+                const position = $(post)
+                    .find('strong')
+                    .not(":contains('Źródło danych:')")
+                    .text();
+                const faculty = $(post)
+                    .find('a')
+                    .get()
+                    .reduce((acc: string[], el) => {
+                        acc.push($(el).text());
+                        return acc;
```
Comment (jzapiorkowski) [inline src/Services/staffScraper.service.ts:29]: can reduce function be swapped for a map?

File: src/Services/staffScraper.service.ts [hunk 3]
```diff
@@ -12,11 +12,23 @@ const facultyMemberScraper = async (
...
+                    }, []);
+                return { position, faculty };
```
Comment (jzapiorkowski) [inline src/Services/staffScraper.service.ts:31]: add empty line before return statement

**Review by jzapiorkowski — APPROVED** [review]
LGTM, just minor changes

**Pull request merged**